### PR TITLE
Add installatievergadering resource

### PR DESCRIPTION
**Pull request by ruben-bottu**

### CODE REVIEW

File: config/migrations/2024/20240313000000-installatievergadering-statussen.ttl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@prefix conceptschemes: <http://lblod.data.gift/concept-schemes/>.
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+@prefix ivs: <http://data.lblod.info/id/InstallatievergaderingStatus/>.
+
+
+# Installatievergadering status concept scheme
+conceptschemes:installatievergadering-status
+  rdf:type 
+    skos:ConceptScheme,
+	  skos:Concept,
```
Comment (Rahien) [inline config/migrations/2024/20240313000000-installatievergadering-statussen.ttl:14]: this should not be a skos:Concept

File: config/resources/verkiezingen.lisp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+             (bestuursorgaan :via ,(s-prefix "ext:hasBestuursorgaanInDeTijd")
+                             :as "bestuursorgaan-in-de-tijd"))
+  :resource-base (s-url "http://data.lblod.info/id/Installatievergaderingen/")
+  :features '(include-uri)
+  :on-path "installatievergaderingen")
+
+(define-resource installatievergadering-status ()
+  :class (s-prefix "ext:InstallatievergaderingStatus")
+  :properties `((:label :string ,(s-prefix "skos:prefLabel"))
+                (:scope-note :string ,(s-prefix "skos:scopeNote")))
```
Comment (Rahien) [inline config/resources/verkiezingen.lisp:15]: i don't think we're interested in the scope note here.

File: config/resources/verkiezingen.lisp [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+             (verkiezingsresultaat-gevolg-code :via ,(s-prefix "mandaat:gevolg")
+                                               :as "gevolg"))
+  :resource-base (s-url "http://data.lblod.info/id/verkiezingsresultaten/")
+  :features '(include-uri)
+  :on-path "verkiezingsresultaten")
+
+(define-resource verkiezingsresultaat-gevolg-code ()
+  :class (s-prefix "ext:VerkiezingsresultaatGevolgCode")
+  :properties `((:label :string ,(s-prefix "skos:prefLabel"))
+                (:scope-note :string ,(s-prefix "skos:scopeNote")))
```
Comment (Rahien) [inline config/resources/verkiezingen.lisp:76]: same, no need for the scope note

**Review by Rahien — CHANGES_REQUESTED** [review]
just some things to remove

File: config/migrations/2024/20240313000000-installatievergadering-statussen.ttl (hunk 1 above)
Comment (ruben-bottu) [reply config/migrations/2024/20240313000000-installatievergadering-statussen.ttl:14]: Ah damn, I also did it like this in decretaal-bestuursorgaan-codes.ttl

**Review by Rahien — APPROVED** [review]

**Pull request merged**